.changeset/lazy-waves-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': patch
+---
+
+'wrapperProps', 'iconProps', 'labelProps', 'subLabelProps' added to RadioTile to individually customize each part of the component.'
```

Comment:
last comment: changeset should mention the breaking change where some props have moved

```suggestion
---
'@itwin/itwinui-react': major
---

`RadioTile`'s `className` and `style` props will now be applied on the `<input>` element instead of the wrapper. Added `wrapperProps`, `iconProps`, `labelProps`, `subLabelProps` to individually customize each part of the component.
```

Reply:
Done